packages/example-react/project/app/Request/Request.js

```diff
@@ -1,39 +1,36 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { AvForm } from 'availity-reactstrap-validation';
 import { Button, Card, CardBody } from 'reactstrap';
 import { navigate } from '@reach/router';
 
 import { Agreement, Patient, Provider } from './components';
-import { Footer, Header } from '../components';
+import { Footer, Header } from '~/shared';
 
-class AuthorizationsRequest extends Component {
-  submit = () => {
+export default () => {
```

Comment:
Not a fan of this change.

Maybe instead you could use 
```javascript
export default Request = () => {
```
So there is a named component that matches the file name.

Reply:
> Not a fan of this change.
> 
> Maybe instead you could use
> 
> ```js
> export default Request = () => {
> ```
> 
> ```js
> export default const Request = () => {
> ```
> 
> So there is a named component that matches the file name.

I believe `eslint` will actually pick up that first suggestion as an `undef` error but you can still achieve the same thing by first just assigning the component to a const and exporting. We do that alot mostly in just cases where functional components need propTypes and/or other logic like injecting global store state before exporting. If this were changed it would look something like:

```jsx
const Request = () => {};
export default Request:
```

Its a matter of preference and in most cases I at least choose the current one because its more shorthand since you already know the folder you are inside of. I can't speak for everyone but probably not worth the effort to change.